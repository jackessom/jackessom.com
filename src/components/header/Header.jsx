import React from 'react';

import './header.css';

function Header(props) {
  return (
    <div className="header">
      <h1>Hello, {props.name}</h1>
    </div>
  );
}

Header.propTypes = {
  name: React.PropTypes.string,
};

Header.defaultProps = {
  name: 'World',
};

export default Header;
