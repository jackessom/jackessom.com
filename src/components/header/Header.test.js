import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('Header shows name', () => {
  const component = renderer.create(
    <Header />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
