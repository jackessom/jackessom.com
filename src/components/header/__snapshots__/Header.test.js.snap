// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header shows name 1`] = `
<div
  className="header"
>
  <h1>
    Hello, 
    World
  </h1>
</div>
`;
